pkg/ccl/kvccl/kvfollowerreadsccl/followerreads_test.go

```diff
@@ -674,19 +687,51 @@ func TestOracle(t *testing.T) {
 			st := cluster.MakeTestingClusterSettings()
 			kvserver.FollowerReadsEnabled.Override(ctx, &st.SV, !c.disabledFollowerReads)
 
-			o := replicaoracle.NewOracle(followerReadOraclePolicy, replicaoracle.Config{
-				NodeDescs:  nodes,
-				Settings:   st,
-				RPCContext: rpcContext,
-				Clock:      clock,
-				HealthFunc: func(roachpb.NodeID) bool { return true },
-			})
+			o := replicaoracle.NewOracle(followerReadOraclePolicy, cfg(st))
 
 			res, _, err := o.ChoosePreferredReplica(ctx, c.txn, desc, c.lh, c.ctPolicy, replicaoracle.QueryState{})
 			require.NoError(t, err)
 			require.Equal(t, c.exp, res)
 		})
 	}
+
+	t.Run("bulk", func(t *testing.T) {
```

Comment:
can we assert the expected values of ignoreMisplannedRanges in the test below too?